reducers/views/rhs.js

```diff
@@ -173,8 +173,6 @@ function isSidebarOpen(state = false, action) {
         return false;
     case ActionTypes.OPEN_LHS:
         return false;
-    case TeamTypes.SELECT_TEAM:
```

Comment:
I think that would make sense to do, although it would require changing everywhere in here and in mattermost-redux that dispatches a SELECT_TEAM action which is a bit annoying.

Reply:
It would be nicer, but doing it this way would probably be a much more complicated and risky change. 
I did try but I ran into an issue where the previous team isn't stored in redux, it's in local storage of the webapp so I can't access it from mattermost-redux. 

I could spend a bunch of time changing that but I am not sure it's worth it. Thoughts?

Reply:
The local storage stuff is indeed challenging. Ideally, everything should be in Redux, with some stuff persisted to/from the local storage transparently, without the webapp ever having to directly access it. A future refactor, I suppose.